src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -551,13 +555,22 @@ public bool IsEnabled()
         {
             if (Node.IsFrozen || !Node.IsVisible) return false;
 
-            if (Node.CachedValue == null || Node.CachedValue.IsNull)
+            if (IsNodeValueNull())
             {
                 return false;
             }
 
             return active;
         }
+
+        public bool IsNodeValueNull()
+        {
+            if (Node.CachedValue == null || Node.CachedValue.IsNull)
```

Comment:
you could have simply returned `return Node == null || Node.CachedValue == null || Node.CachedValue.IsNull;`